internal/config/subnet/api-key.go

```diff
@@ -49,7 +49,7 @@ type Config struct {
 	APIKey string `json:"api_key"`
 
 	// The HTTP(S) proxy URL to use for connecting to SUBNET
-	Proxy string `json:"proxy"`
+	ProxyURL *url.URL `json:"proxy_url"`
```

Comment:
you can our project here *xnet.URL - does a more stricter parsing. and also parse the URL as `xnet.ParseHTTP()`

Reply:
Done